sdk/textanalytics/ci.yml

```diff
@@ -2,41 +2,43 @@
 # This file is generated automatically and any changes will be lost.
 
 resources:
-    repositories:
-        - repository: azure-sdk-build-tools
-          type: git
-          name: internal/azure-sdk-build-tools
-        - repository: azure-sdk-tools
-          type: github
-          name: Azure/azure-sdk-tools
-          endpoint: azure
+  repositories:
+    - repository: azure-sdk-build-tools
+      type: git
+      name: internal/azure-sdk-build-tools
+      ref: refs/tags/azure-sdk-build-tools_20200514.1
+    - repository: azure-sdk-tools
+      type: github
+      name: Azure/azure-sdk-tools
+      endpoint: azure
+      ref: refs/tags/azure-sdk-tools_20200514.1
 
 trigger:
-    branches:
-        include:
-            - master
-            - hotfix/*
-            - release/*
-    paths:
-        include:
-            - sdk/textanalytics/
+  branches:
```

Comment:
Do the incorrect indentations here mean this pipeline wasn't properly triggering?

Reply:
Nope, its was using tabs rather than spaces but it stillworks